.gitignore

```diff
@@ -2,6 +2,8 @@ bin
 gen
 target
 logs
+.idea
```

Comment:
IDE specific files (except eclipse because it's the official build platform) should be handled in `.git/info/exclude` because the number of IDEs and editor is endless and thus `.gitignore` might have an endless length one day.